_raml/apps/carts/carts.raml

```diff
@@ -104,6 +104,21 @@ post:
               example: !include examples/post_carts_cartid_lineitem_response.json
               schema: !include schema/post_carts_cartid_lineitem.schema.json
 
+  /multi-line-items:
+    displayName: Multiple line items
+    post:
+      description: Create product line items in a cart. *epages6 *epagesNow
+      body:
+        application/json:
+          example: !include examples/post_carts_cartid_multilineitem_request.json
+          schema: !include schema/post_carts_cartid_multilineitem_request.schema.json
+      responses:
+        201:
+          body:
+            application/json:
+              example: !include examples/post_carts_cartid_lineitem_response.json
```

Comment:
This response does not match to the request as three line items are mentioned there. The response should thus also show these three items and not only one...

Reply:
ok